kornia/geometry/transform/crop.py

```diff
@@ -13,6 +13,7 @@
     "bbox_to_mask",
     "infer_box_shape",
     "validate_bboxes",
+    "bbox_generator"
```

Comment:
we should consider having a dedicated module for bboxes. `kornia.geometry.transform.boxes`. I'll open an issue with feature request.